docs/style_guide.adoc

```diff
@@ -0,0 +1,152 @@
+
+When linking to other content within the site, we like to take steps to make it as obvious as possible without ruining the flow of the doc. Here are some guidelines for formatting your links:
+
+1. Include the title of the guide in the link text. For example...
++
+----
+link:/guides/openshift-ha-installation{outfilesuffix}[Installing a Highly Available OpenShift Cluster]
+----
++
+rather than
++
+----
+link:/guides/openshift-ha-installation{outfilesuffix}{outfilesuffix}[click here!]
+----
++
+2. Provide some context as to why you are providing this link. For example...
++
+----
+The link:/articles/openshift-load-balancing{outfilesuffix}[External Load Balancing for OpenShift] provides an introduction to the strategies that can be employed within OpenShift.
+----
++
+3. Use a NOTE callout when there's no natural flow.
++
+----
+NOTE: The link:/articles/load_balancing{outfilesuffix}[External Load Balancing for OpenShift] provides an introduction to the strategies that can be employed within OpenShift.
+----
+
+=== Include a "What's Next?" section to encourage further reading
+
+We like to conclude our documents with a "call to action" or something of that sort. Whenever applicable, think about ending your doc with a "What's Next?" section with links out to further suggested playbooks guides, or other documents when a playbook guide doesn't yet exist (if you think it SHOULD exist, please link:https://github.com/rhtconsulting/openshift-playbooks/issues/new[Open an Issue]).
+
+== Content Principles
+
+All written content should follow these principles:
+
+* **Approachability:** Don't assume reader has prior knowledge
+* **Brevity:** Keep it simple, link to outside content for deeper dives
+* **Curation:** Amplify community best practices vs. any individual's point of view
+
+Content should maintain a light-hearted, but wise (think classy, not overly excited) tone. Open source is fun! Readers should feel inspired, not discouraged, by the tone of your writing, and they should trust you to help them get started.
+
+== Mentions
+
+When referring to people that use GitHub, use @mentions of their username instead of their full name.
+
+- :smile: As @sabre1041 put it...
+- :cry: As [Andrew Block](https://github.com/sabre1041) put it...
+
+When referring to a project on GitHub, link to the repository so others can dive deeper, if they choose.
+
+- :smile: @JaredBurck took a similar approach to [Dat](https://github.com/datproject/dat)...
+- :cry: @JaredBurck took a similar approach to Dat...
+
+== Capitalization
+
+The domain "uncontained.io" is not capitalized when referring to the "uncontained.io guides", except at the beginning of a sentence.
+
+- :smile: Welcome to uncontained.io!
+- :smile: The uncontained.io site is meant to...
+- :cry: The goal of Uncontained.io is to...
+- :cry: The mission of UnContained.io is to...
+
+The word "guides" is not capitalized when referring to the "uncontained.io guides", just like saying "the guide" or "this guide".
```

Comment:
This looks like an artifact left over from the GitHub version. We don't use "guides" in the name of the site - it's just "uncontained.io", not "uncontained.io guides". Can we remove this section?

Reply:
I see the changes made and approve